Calculator

```diff
@@ -1 +1,50 @@
+package com.company;
 
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+        Main main = new Main();
+        System.out.println("Enter number: ");
+        double number = input.nextDouble();
+        System.out.println("Enter degree of number: ");
+        double degreeOfNumber = input.nextDouble();
+        System.out.println("Result is: ");
+        if (degreeOfNumber == 2) { number = main.squareOfNumber(number); }
+        else if (degreeOfNumber == 3) { number = main.cubeNumber(number); }
+        else {
+        number = main.result(number, degreeOfNumber); }
+        System.out.println(number);
+    }
+
+    public double result(double number, double degreeOfNumber) {
```

Comment:
rename this method, for example power or pow.

Reply:
you can make this method simply. Create two conditions, for positive and negative number.
if (degreeOfNumber < 0) {
            for (int i = degreeOfNumber; i < 0; i++) {
                   result *= number;
            }
            result = 1 / result;
        } else {
            for (int i = 0; i < degreeOfNumber; i++) {
                result *= number;
            }
        }
        return result;